CHANGELOG.md

```diff
@@ -197,6 +198,51 @@ Example:
 │                        │         │        │            │             │                 │ Identity Roles       │  three, two,one        │
 ╰────────────────────────┴─────────┴────────┴────────────┴─────────────┴─────────────────┴──────────────────────┴────────────────────────╯
 ```
+### Allow dynamic modification of enrollment durations
+#### Identity Enrollment Duration
+Setting the environment variable `ZITI_EDGE_IDENTITY_ENROLLMENT_DURATION` to some value **in minutes** will override the default identity enrollment duration configuration
```

Comment:
i kinda feel like it would be advantageous to have all the env vars to use the same unit. maybe that's a good issue to file?